dvc/repo/add.py

```diff
@@ -49,7 +52,21 @@ def add(repo, targets, recursive=False, no_commit=False, fname=None):
 
             stages = _create_stages(repo, sub_targets, fname, pbar=pbar)
 
-            repo.check_modified_graph(stages)
+            try:
+                repo.check_modified_graph(stages)
+            except OverlappingOutputPathsError as e:
+                msg = (
+                    "Cannot add '{out}', because it is overlapping with other "
+                    "DVC tracked output: '{parent}'.\nTo include '{out}' in "
```

Comment:
It is better to move a line down if you have `\n`. E.g.
```
"Cannot add '{out}', because it is overlapping with other "
"DVC tracked output: '{parent}'.\n"
"To include '{out}' in ..."
```
much easier to comprehend that way. Or was it black who did that?

Reply:
nah, black gets in the way only when lines do not match, I probably did that to save some lines :P